resources/base.scss

```diff
@@ -22,6 +22,7 @@ a {
 
     &:hover {
         color: #0645ad;
+        cursor: pointer;
```

Comment:
I'm not strictly against this, but it seems a bit odd to include in this PR. Can we set the cursor on the one link that has problems (or are there multiple)?